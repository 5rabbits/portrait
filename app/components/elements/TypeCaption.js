import React, { PureComponent } from 'react';

export default class TypeCaption extends PureComponent {
  render() {
    return (
      <span className="type-caption"></span>
    );
  }
} 
