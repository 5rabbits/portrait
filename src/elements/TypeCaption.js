import React from 'react'

const TypeCaption = () =>
  <span className="type-caption" />

export default TypeCaption
