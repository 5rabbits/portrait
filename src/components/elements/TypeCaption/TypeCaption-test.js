import { React, expect, mount } from 'test/helper'
import TypeCaption from 'elements/TypeCaption'

describe('TypeCaption', () => {
  it('should render with minimum props', () => {
    const component = mount(<TypeCaption />)

    expect(component).to.be.ok
  })
})
