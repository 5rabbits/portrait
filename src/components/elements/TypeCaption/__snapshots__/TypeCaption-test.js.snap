// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TypeCaption should render with minimum props 1`] = `
<TypeCaption>
  <span
    className="type-caption"
  />
</TypeCaption>
`;
